man/systemd.exec.xml

```diff
@@ -2525,6 +2525,19 @@ RestrictNamespaces=~cgroup net</programlisting>
         <xi:include href="version-info.xml" xpointer="v258"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>BPFDelegateCommands=</varname></term>
+
+        <listitem><para>Accepts a list of BPF commands to delegate or <literal>any</literal> to allow everything,
```

Comment:
`any` -> `all` is better?
Please mention the list of possible values of BPF commands.

Reply:
"any" is consinstent with the kernel API, which uses "any" to delegate all types

Reply:
OK.